### ISSUE TITLE
Crashes when used with fabric 1.20.4

### ISSUE THREAD

**Issue by InvertedCandy14** [Draylar/identity#633]
idk it doesnt even open with the mod installed does it have to be architectury 1.20.1 cuz if so then im kinda stupid

**Comment by ToCraft** [comment]
Well, this mod isn't for 1.20.2+ but for 1.20.1 only.

**Comment by InvertedCandy14** [comment]
yeah i know but why do all the good mods have to be on 1.20.1 and not the latest version

**Comment by ToCraft** [comment]
You _could_ try my fork of identity...
It's actually for the latest version, including neforge.
https://www.curseforge.com/minecraft/mc-mods/remorphed

**Comment by InvertedCandy14** [comment]
ya but is there abilities

**Comment by ToCraft** [comment]
Yeah, of course. There are even more abilities and features :D

**Comment by InvertedCandy14** [comment]
cool and all but... theres no sounds (e.g. ender dragon flapping and stuff) and when falling as a chicken the wings dont flap

**Comment by ToCraft** [comment]
Yeah, I'm aware of this issues and they're on my todo-list.

**Comment by ArduFish** [comment]
@ToCraft Whenever I see a zombie or a skeleton or a shulker the server crashes

**Comment by ArduFish** [comment]
Or an endermite or a minecart

**Comment by ToCraft** [comment]
> @ToCraft Whenever I see a zombie or a skeleton or a shulker the server crashes

When you're using Identity, please report it to Draylar instead.
